platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -295,9 +295,21 @@ sep1:
 sep6:
   # Whether to enable SEP-6
   enabled: false
+  # Configures the more_info_url of the transaction response when calling GET /transaction and GET /transactions endpoints.
+  # For details, please refer to https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#transaction-history
+  #   base_uri: is the base_uri used to construct the more_info_url
+  #   jwt_expiration: the timeout in seconds of the JWT returned with the embedded interactive url of the SEP-6 process.
+  #       The jwt secret must be set by the environment variable: secret.sep6.more_info_url.jwt_secret
+  #   txn_fields: is an optional list of transaction fields that will be appended to the jwt of the more_info url. If empty, no fields from transaction will be appended.
+  #   In addition to the txn_fields, the following fields are also added to the query parameters.
+  #       `transaction_id`: the transaction ID
+  #       `token`: the JWT token
+  more_info_url:
+    base_url: http://localhost:8080/sep6/transaction/more_info
```

Comment:
It's a bit weird to have a default base URL since the default points to an endpoint that doesn't exist on the SEP server. I guess this is for consistency with SEP-24 but I don't think there's any problem with removing it from both. What do you think?

Reply:
IMO this acts more like a placeholder. `base_url` is a required field and validation error will be thrown if we left it empty.

Remove it would require us to add them as env var to all the other config under service-runner/src/main/resources/profiles

Reply:
Leaving it empty is almost the same thing as having a default URL since it points to an endpoint that doesn't exist. I honestly think we should make it optional and remove the validation. 

Thoughts @lijamie98 ?